pyresample/geometry.py

```diff
@@ -2109,7 +2111,12 @@ def create_areas_def_legacy(self):
         return area_def_str
 
     def __eq__(self, other):
-        """Test for equality."""
+        """Test for equality.
+
+        Equality considers only the projection coordinates, not the description
+        or projection ID.  In other words, areas are considered equal if they
```

Comment:
I wonder if there is a way to rephrase this to use the word "metadata" or some other word meaning "information that isn't the projection/grid information". Any ideas?